### PR TITLE
VAVFA-8986: Add queries and include for vamc covid status.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Adds expandable alert for facility covid status usage. See:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8986
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8981
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8982
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8983
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9083

Sibling pr for vets-website vamc expandable widget needed for functionality: https://github.com/department-of-veterans-affairs/vets-website/pull/21037

Tugboat data endpoint to use for testing: https://pr9028-sjpg4ebinnl4eooevngpwufok19n4akk.ci.cms.va.gov/
Adding @maxx1128 & @swirtSJW for situational awareness.

## Testing done
Visual / build

## Screenshots
image

image

image

image

## Acceptance criteria
- [ ] Check out `https://github.com/department-of-veterans-affairs/vets-website/pull/21037` and run `yarn watch` (in vets-website)
- [ ] In content-build, run `yarn preview --drupal-address=https://pr9028-sjpg4ebinnl4eooevngpwufok19n4akk.ci.cms.va.gov/ --drupal-user=ethan.teague --drupal-password=drupal8`
- [ ] Visually verify presence and functionality of expandable/collapsible covid status alerts when visiting http://localhost:3002/preview?nodeId=90
- [ ] - [ ] Visually verify presence and functionality of expandable/collapsible covid status alerts when visiting http://localhost:3002/preview?nodeId=1010
- [ ] - [ ] Visually verify presence and functionality of expandable/collapsible covid status alerts when visiting http://localhost:3002/preview?nodeId=2802

### CODE REVIEW

File: src/site/includes/covid-status.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% if fieldSupplementalStatus.entity.name %}
+	<div data-widget-type="expandable-vamc-operating-status-node-{{ entityId }}" facilityid="node-{{ entityId}}" statuslabel="{{ fieldSupplementalStatus.entity.name }}" status="limited" info="
+									{% if fieldSupplementalStatus.entity.description.processed %}
+										{{ fieldSupplementalStatus.entity.description.processed }}
+									{% endif %}
+									{% if fieldSupplementalStatus.entity.description.processed %}
```
Comment (maxx1128) [inline src/site/includes/covid-status.drupal.liquid:6]: I think this one needs to be `fieldSupplementalStatus.entity.fieldGuidance.processed`

Comment (ethanteague) [reply src/site/includes/covid-status.drupal.liquid]: @maxx1128 Thanks for the save, buddy :+1:

File: src/site/layouts/health_care_local_facility.drupal.liquid [hunk 2]
```diff
@@ -39,11 +40,17 @@
...
                   {% if fieldOperatingStatusFacility and fieldOperatingStatusFacility != 'normal' %}
-                    <div class="vads-u-display--inline-block vads-u-margin-bottom--2p5">
+                    <div class="vads-u-display--inline-block vads-u-margin-bottom--1">
                       {% include "src/site/includes/operatingStatusFlagsLinks.drupal.liquid" %}
                     </div>
                   {% endif %}
+
+                  {% assign bottomMargin = '2' %}
+                  {% assign topMargin = '0p5' %}
+                  {% include "src/site/includes/covid-status.drupal.liquid" %}
```
Comment (maxx1128) [inline src/site/layouts/health_care_local_facility.drupal.liquid:52]: @ethanteague do we need to wrap a conditional around this `include` to make sure the COVID status alert doesn't appear if a COVID status hasn't been set?

Comment (ethanteague) [reply src/site/layouts/health_care_local_facility.drupal.liquid:52]: Nah, we're protected by `{% if fieldSupplementalStatus.entity.name %}` in `src/site/includes/covid-status.drupal.liquid`.
Good thinking / good eye, @maxx1128 :+1:

**Review by maxx1128 — APPROVED** [review]

**Pull request merged**